pose_collector:
  clock: 100
virtual_drone_clock: 100
field_cbf: [[0, 1.5], [0,5]] # 衝突回避で避けられるよう、psiよりも一回り大きくとる
central_clock: 20
angle_aware:
  sigma: 0.1
  delta_decrease: 5
  gamma: 0.01
  alpha: 1
  # phi:
  #   range: [[-1, 1],[0, 1], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [1, 1,0.5, 0.78, 6.28]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-1, 1],[0, 1]]
  #   density: [1, 1]
  # phi:
  #   range: [[-6, 6],[-6, 6], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [0.1, 0.1, 0.5, 0.2617993878, 0.2617993878]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-8, 8],[-8, 8]]
  #   density: [0.1, 0.1]
  # phi:
  #   range: [[-1, 1],[-1, 1], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [0.01, 0.01, 0.5, 0.2617993878, 0.2617993878]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-1.3, 1.3],[-1.3, 1.3]]
  #   density: [0.02, 0.02]
  phi:
    range: [[0.3, 1.2], [0.3,4.7],[1.5,1.5], [-1.5707963267948966, -1.0471975511965976],[-3.14159265358979,3.14159265358979]] #xyzvh, v: - [pi/3, pi/2]
    density: [0.05, 0.05, 0.05, 0.05235987755982988, 0.08726646259971647]  #v: pi/60, h: pi/36
  psi:
    range: [[0.1, 1.4], [0.1, 4.9]]
    density: [0.05, 0.05]
    npy_name: psi_temp.npy
agents:
  agent_manager_clock: 20
  u_max: 0.2
  collision_distance: 0.9
  camera_sensor_rad: 0.7895822394
  kp_z: 0.1
  ref_z: 1
  kp_yaw: 1
  ref_yaw: 0
